pkg/provider/azure.go

```diff
@@ -1229,9 +1229,11 @@ func (az *Cloud) updateNodeCaches(prevNode, newNode *v1.Node) {
 		switch {
 		case !isNodeManagedByCloudProvider:
 			az.excludeLoadBalancerNodes.Insert(newNode.ObjectMeta.Name)
+			klog.V(4).Infof("excluding Node %q from LoadBalancer because it is not managed by cloud provider", newNode.ObjectMeta.Name)
```

Comment:
This will add a lot of logs for large clusters. Can we change it to 6?

Reply:
I chose 4 because I think the log level should be same with the below logic:
```
		for _, address := range getNodePrivateIPAddresses(newNode) {
			if az.nodePrivateIPs[newNode.Name] == nil {
				az.nodePrivateIPs[newNode.Name] = sets.New[string]()
			}

			klog.V(4).Infof("adding IP address %s of the node %s", address, newNode.Name)
			az.nodePrivateIPs[newNode.Name].Insert(address)
		}
```
Shall we make all of them 6?